nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,85 @@
+---
+title: SQL Server Transport usage guidance
+summary: Usage guidance for SQL Server transport
+reviewed: 2016-03-22
+tags:
+- SQL Server
+---
+
+The SQL Server transport implements a message queueing mechanism on top of Microsoft SQL Server. It provides support for sending messages over [SQL Server](http://www.microsoft.com/en-au/server-cloud/products/sql-server/) tables. It does **not** make any use of ServiceBroker, a messaging technology built into the SQL Server.
+
+
+## How it works
+
+The SQL Server transport is a hybrid queueing system which is neither store-and-forward (like MSMQ) nor a broker (like RabbitMQ). It treats the SQL Server as a storage infrastructure for the queues while the queue implementation logic is executed inside the client process.
+
+
+### Advantages of SQL Server Transport
+
+ * No additional licensing and training costs, as majority of Microsoft stack organizations already have SQL Server installed and have knowledge required to run it
+ * Great general-purpose tooling (SSMS)
+ * Throughput on par with MSMQ
+ * Scaling out of processing is simple because of support for competing consumers at the queue level
+
+## Usage scenarios - deployment size
+
+### Tiny
+
+The SQL Server transport is an ideal choice for extending an existing web application with asynchronous processing capabilities as an alternative for tedious batch jobs that tend to quickly get out of synch with the main codebase. Assuming the application already uses SQL Server as a database, this scenario does not require any additional infrastructure.
+
+The queue tables are hosted in the same catalog as business data and the NServiceBus runtime can be hosted in the web worker. In some cases there is a need for a separate process for NServiceBus. In most cases these systems consists of 1 to 3 non-scaled out endpoints. Because they consist of a single logical service or bounded context, there is usually no need to create separate schemas for the queues.
```

Comment:
I wanted to provide some number to give users some kind of hint when they should be thinking about switching to higher tier/complexity models: from tiny to small (introduce schemas) and then to medium/large (move all transport to a dedicated catalog).